modules/exploits/linux/http/mvpower_dvr_shell_exec.rb

```diff
@@ -60,6 +60,7 @@ def check
     begin
       fingerprint = Rex::Text::rand_text_alpha(rand(10) + 6)
       res = send_request_cgi(
+        'method' => "GET",
         'uri' => "/shell?echo+#{fingerprint}",
```

Comment:
The pleasure is all mine, I've learnt a lot working with this. By the way, the (method => "GET")  wasn't there from before. I added it, I hope you've reviewed it and it won't cause any problem.